lib/event_manager.rb

```diff
@@ -1,9 +1,8 @@
+require "csv"
 puts "EventManager Initialized!"
 
-lines = File.readlines "event_attendees.csv"
-lines.each_with_index do |line, index|
-  next if index == 0
-  columns = line.split(",")
-  name = columns[2]
+contents = CSV.open "event_attendees.csv", headers: true, header_converters: :symbol
```

Comment:
I would, in this case, probably prefer them. I went with their method here - not even thinking that I could add them. Agreed in this case it would make it much easier to read.